### DISCUSSION TITLE
Support RSS

### DISCUSSION THREAD

**Discussion by Muzych** [withastro/starlight#1549] [Feature Requests]
### What version of `starlight` are you using?

0.18.0

### What is your idea?

RSS Integration.

### Why is this feature necessary?

Some people are tired of the information cocoon brought by recommendation algorithms. They are more willing to actively select the information they receive than to passively receive it. Therefore, I think that as a document platform, it is necessary to integrate RSS functions.

### Participation

- [ ] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Hi @Muzych! Thanks for the suggestion.

I'm curious how you think this might look for Starlight pages which are not timestamped in the way RSS posts usually are? I can imagine RSS support in the [`starlight-blog`](https://github.com/HiDeoo/starlight-blog) plug-in would make a lot of sense, but I'm less clear what it would mean for an undated docs page.

**Reply by Muzych** [reply]
Yes, you are right. It is more appropriate to integrate the RSS function into more time-sensitive documents such as blogs.
